README.md

```diff
@@ -1 +1,8 @@
-# best-repo-ever
\ No newline at end of file
+# best-repo-ever
+This is new content added with Notepad++
+Second Line 
+When you work locally, your files exist in one of four states. 
+They are either untracked, modified, staged, or committed. 
+Git tracks these files, and keeps track of your history by organizing 
+your files and changes in three trees. 
+This is last line in first editing. 
```

Comment:
This line comment is to test write tab in files changed tab in pull requests tab in the repository best-repo-ever